WordPress/wordpress-importer
# Move build bundle to `build/`

This PR moves the build target to `build/` and ignores that directory so that we're not checking in built changes on each update.

### Testing

- Checkout this branch
- Make sure there's no `src/build.js` file
- Run `npm run build`
- Make sure the UI works just as before
